.gitignore

```diff
@@ -5,3 +5,4 @@ docs/_build
 build
 data
 dist/
+.DS_Store
```

Comment:
This should be added on your global .gitignore

Reply:
I understand, but why not?

Reply:
Just because that's a auto generated and not necessary.